install

```diff
@@ -194,10 +194,9 @@ end
 # return the shell profile file based on users' preference shell
 def shell_profile
   case ENV["SHELL"]
-  when %r{/(ba)?sh} then "~/.bash_profile"
-  when %r{/zsh} then "~/.zshrc"
-  when %r{/ksh} then "~/.kshrc"
-  else "~/.bash_profile"
+  when %r{/bash} && File.writable?("~/.bash_profile") then "~/.bash_profile"
```

Comment:
Please change `.writable?` to `.readable?`. If it exists with read-only perms, that's the file that `bash` will be reading, and the one that we should update. It'd be better to get an error saying we don't have perms to modify `~/.bash_profile` than to modify the wrong file.

Reply:
Please change `%r{/bash}` to `%r{/bash$}`